src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
It looks like you're extending src/interfaces/social.ts#UserStatus from the uProxy repo, is that right?